### PR TITLE
Add Send to Peer-To-Peer Filesharing.

### PR DESCRIPTION
**Pull request by kokomo123**
This is basically a fork of Mozilla's Firefox Send thats now discontinued/archived, just without the branding and everything. It was made as a community effort to keep Firefox Send alive in a way. So you can legally self-host it. A friend recommended this to me, so i felt like it's appropriate to add it back into Awesome-Selfhosted. Hope this is acceptable. Couldn't find a website in the repo so feel free to make edits if you can find a main demo one or a main website to it. 

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [X] Your submissions are formatted according to the following requirements (`Demo` and `Clients` are optional, don't add a duplicate `Source code` link if it is the same as the main link):

``- [Name](http://homepage/) - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

- [ ] If your additions depend on proprietary services outside the user's control, they must be marked `⚠`.

``- [Name](http://homepage/) `⚠` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

- [ ] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and marked `⊘ Proprietary`:

``- [Name](http://homepage/) `⊘ Proprietary` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``

- [X] Your additions are ordered alphabetically.
- [ ] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics) (analytics), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [X] The `Language` tag is the main server-side requirement for the software - don't include frameworks/specifc dialects.
- [X] Any licenses you have added are in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [ ] Any category you are creating has the minimum requirement of 3 items. If not, your addition may be inserted into `Misc/Other`.
- [X] Any software project you are adding to the list is actively maintained.
- [X] The pull request title is "Add aaa to bbb" for adding software aaa to section bbb. Do not call it "Update README.md".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -721,6 +721,7 @@ See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems
 - [cloud-torrent](https://github.com/jpillora/cloud-torrent) - Torrent Web Client with HTTP retrievable or streamable downloaded files. `AGPL-3.0` `Go`
 - [Dat Project](https://datproject.org) - Powerful decentralized file sharing applications built from a large ecosystem of modules. ([Source Code](https://github.com/datproject)) `MIT` `Nodejs`
 - [FilePizza](http://file.pizza/) - Peer-to-peer file transfers in your browser. ([Source Code](https://github.com/kern/filepizza)) `BSD-3-Clause` `Nodejs`
+- [Send](https://gitlab.com/timvisee/send) - A file sharing experiment which allows you to send encrypted files to other users. `MPL-2.0` `JavaScript`
```
Comment (n8225) [inline README.md:724]: Language should be Nodejs

Additionally you could add https://gitlab.com/timvisee/send/-/tree/master/android as a client.

**Review by n8225 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kokomo123) [reply README.md]: Done. Hope this helps ^^

**Comment by kokomo123** [comment]
Oops. Accidentally did the wrong thing. Making a new commit now. Sorry :/

**Comment by kokomo123** [comment]
Seems the PR is failing. Not sure what i did wrong, but regardless i think its A-OK. Feel free to request more changes, sorry i keep filling up the commits in here haha.

File: README.md [hunk 2]
```diff
@@ -721,6 +721,7 @@ See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems
 - [cloud-torrent](https://github.com/jpillora/cloud-torrent) - Torrent Web Client with HTTP retrievable or streamable downloaded files. `AGPL-3.0` `Go`
 - [Dat Project](https://datproject.org) - Powerful decentralized file sharing applications built from a large ecosystem of modules. ([Source Code](https://github.com/datproject)) `MIT` `Nodejs`
 - [FilePizza](http://file.pizza/) - Peer-to-peer file transfers in your browser. ([Source Code](https://github.com/kern/filepizza)) `BSD-3-Clause` `Nodejs`
+- [Send](https://gitlab.com/timvisee/send) - A file sharing experiment which allows you to send encrypted files to other users. [Client](https://gitlab.com/timvisee/send/-/tree/master/android) `MPL-2.0` `Nodejs`
```
Comment (nodiscc) [inline README.md:724]: `([Clients](https://gitlab.com/timvisee/send/-/tree/master/android))`

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (kokomo123) [reply README.md]: Fixed it up for good now

**Review by n8225 — APPROVED** [review]

**Review by nodiscc — APPROVED** [review]

**Pull request merged**